### PR TITLE
Add debugs for CPU Quota flake

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@bparees PTAL

File: test/extended/fixtures/build-quota/.s2i/bin/assemble [hunk 1]
```diff
@@ -5,11 +5,17 @@ echo -n MEMORY= && cat /sys/fs/cgroup/memory/memory.limit_in_bytes
 echo -n MEMORYSWAP= && cat /sys/fs/cgroup/memory/memory.memsw.limit_in_bytes
 
 if [ -e /sys/fs/cgroup/cpuacct,cpu ]; then
+	qouta=$(</sys/fs/cgroup/cpuacct,cpu/cpu.cfs_quota_us)
```
Comment (bparees) [inline test/extended/fixtures/build-quota/.s2i/bin/assemble]: quota

**Comment by bparees** [comment]
[testextendedonly][extended:core(s2i build with a quota)]

**Comment by mrunalp** [comment]
@bparees updated

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
[merge]

**Pull request merged**